EzMap.Domain/Models/PoiCollectionConfiguration.cs

```diff
@@ -11,5 +11,6 @@ public void Configure(EntityTypeBuilder<PoiCollection> builder)
         builder.ToTable(nameof(PoiCollection)).HasKey(x => x.Id);
         builder.Property(x => x.Name);
         builder.Property(x => x.Description);
+        builder.Property(x => x.DefaultViewType);
```

Comment:
I'm not sure if this is buildable, the property name is [ViewType](https://github.com/Ez-Map/ezmap-poc/pull/4/files#diff-0da67a4d733d7601a1c6ea6c4d3b879f0156f52a0976c188421eb76a53e45cc8R15) while this says `DefaultViewType`